### PR TITLE
[Azure Stack] Removed Operations endpoint

### PR DESCRIPTION
**Pull request by deathly809**
### Latest improvements:

I think your online Linter finished?  It said no errors, also Autorest reported no errors as well.

Anyways, this endpoint is not an RP and an operations endpoint was added by mistake. I removed all references to Operations endpoint and added a suppression.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [X] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by deathly809** [comment]
lol the link the bot sent me doesn't work,

image

File: specification/azsadmin/resource-manager/user-subscriptions/readme.md [hunk 1]
```diff
@@ -35,6 +35,9 @@ directive:
     where:
       - $.paths["/subscriptions/{subscriptionId}"].put
 
+  - suppress: R3023
+    reason: No operations endpoint as not ARM resource provider.
```
Comment (sergey-shandar) [inline specification/azsadmin/resource-manager/user-subscriptions/readme.md]: What's the reason for this suppression? Should we have ARM approval for this? Is it for a specific place? In this case, should we use `where` and `from`.

Comment (deathly809) [reply specification/azsadmin/resource-manager/user-subscriptions/readme.md]: There is no operations endpoint in ARM.  This is almost the same as the Microsoft.Subscription spec but we have different objects.

Comment (KrisBash) [reply specification/azsadmin/resource-manager/user-subscriptions/readme.md]: This suppression is OK from ARM's perspective.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**